rclcpp/resource/logging.hpp.em

```diff
@@ -45,61 +45,65 @@
 #endif
 
 @{
-from rcutils.logging import feature_combinations
-from rcutils.logging import get_macro_parameters
-from rcutils.logging import get_suffix_from_features
-from rcutils.logging import severities
```

Comment:
Why remove all these imports and update their usage?

Reply:
I'd say it's a leftover from a previous commit. +1 to rollback these changes.

Reply:
I assume this is resolved? (please resolve comments if they are)